actix-http/src/encoding/decoder.rs

```diff
@@ -8,10 +8,13 @@ use std::{
 };
 
 use actix_rt::task::{spawn_blocking, JoinHandle};
+#[cfg(feature = "compress-br")]
```

Comment:
Totally nitpicky and a matter of taste, but I personally find import sequences combined with a lot of `cfg(...)` hard to read. I think it's better to separate `compress-*` imports with an empty line, so we have a more or less clean sequence of unconditional imports and then a sequence of imports guarded by `cfg(...)`.

Reply:
I second that. An empty line makes it much easier to find.

Reply:
Indeed it looks better.

I have update the import related to compression but there is still other import not related to `compress-*` which do not have empty line before.

Would you like I update them in this PR ? create a PR for this ? or just let them as it ?